package.json

```diff
@@ -3,6 +3,11 @@
   "version": "0.5.0",
   "description": "Modern Build System",
   "main": "dist/index.js",
+  "files": [
+    "bin/",
+    "dist/",
+    "LICENSE"
+  ],
```

Comment:
this is an opposite to `.npmignore` file, so only `bin/`, `dist/`, `LICENSE ` and some default npm stuff will be published and then installed by users.